data-import/src/main/resources/folijet/data-import/features/create-order-line-with-vrn.feature

```diff
@@ -0,0 +1,51 @@
+Feature: Create order line
```

Comment:
Hello @KyryloShulzhenko, could you please move this .feature file to the resources/folijet/data-import/global directory where reusable .feature files are located